dimod/constrained/constrained.py

```diff
@@ -1472,7 +1471,7 @@ def count(qm: Union[QuadraticModel, BinaryQuadraticModel]) -> int:
 
         return n
 
-    def relabel_constraints(self, mapping: Mapping[Hashable, Hashable]):
+    def relabel_constraints(self, mapping: Mapping[collections.abc.Hashable, collections.abc.Hashable]):
```

Comment:
```suggestion
    def relabel_constraints(self, mapping: collections.abc.Mapping[collections.abc.Hashable, collections.abc.Hashable]):
```

Reply:
There are a few more in the `CQMToBQMInverter` class.

Reply:
Think I got them all in 49f92ba. Gosh I hate Python's `typing`, especially how unstable it's been over the last few versions.